### ISSUE TITLE
totalopenstation-cli-parser: TypeError: startswith first arg must be bytes or a tuple of bytes, not str

### ISSUE THREAD

**Issue by steko** [totalopenstation/totalopenstation#84]
When calling the command line program with the `--help` flag:

```
$ totalopenstation-cli-parser.py --help
Traceback (most recent call last):
  File "/home/steko/Scaricati/topsenv/bin/totalopenstation-cli-parser.py", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/home/steko/Scaricati/totalopenstation/scripts/totalopenstation-cli-parser.py", line 39, in <module>
    parser = OptionParser(usage=usage)
  File "/usr/lib/python3.7/optparse.py", line 1191, in __init__
    self.set_usage(usage)
  File "/usr/lib/python3.7/optparse.py", line 1270, in set_usage
    elif usage.lower().startswith("usage: "):
TypeError: startswith first arg must be bytes or a tuple of bytes, not str
```

The companion `totalopenstation-cli-connector` is called without showing a similar issue.

**Comment by steko** [comment]
This is caused by calling `lgettext`, which returns byte strings. 

It must be replaced by `gettext` everywhere we call it.

**Comment by steko** [comment]
See https://bugs.python.org/issue29755 for a more detailed explanation.

**Comment by steko** [comment]
The only other place where this is called is `totalopenstation-gui.py`.

**Comment by steko** [comment]
Fixed with 118d15987a1b4cb5752e576e795e4ee516ab415f.
